modules/core/src/main/scala/dev/profunktor/redis4cats/effect/RedisMonad.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2018-2021 ProfunKtor
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.profunktor.redis4cats.effect
+
+import cats.effect._
+import dev.profunktor.redis4cats.connection.{ RedisClient, RedisClusterClient, RedisURI }
+import dev.profunktor.redis4cats.config.Redis4CatsConfig
+import io.lettuce.core.ClientOptions
+import scala.annotation.implicitNotFound
+
+@implicitNotFound("Missing instances for either cats.effect.Async or dev.profunktor.redis4cats.effects.Log")
```

Comment:
I'd mention that this is for `RedisMonad`... but maybe it's already part of the error message. If it is, disregard

Reply:
Hmm let me try again, it happens when you try to summon an implicit `RedisMonad[F]`, I'll double check.

Reply:
Improved the message 👍🏾 

```scala
[error] /home/gvolpe/workspace/redis4cats/modules/examples/src/main/scala/dev/profunktor/redis4cats/RedisStringsDemo.scala:39:16: RedisMonad instance not found. You can summon one by having instances for either cats.effect.Async and dev.profunktor.redis4cats.effects.Log in scope
[error]     commandsApi[IO]
[error]                ^
```